tsup.config.ts

```diff
@@ -0,0 +1,14 @@
+import { defineConfig } from 'tsup';
+
+export default defineConfig({
+  clean: true,
+  dts: true,
+  entryPoints: ['src/index.ts'],
+  external: ['react', 'react-dom', '@contentful/f36-*'],
+  format: ['cjs', 'esm'],
+  minify: true,
+  platform: 'browser',
+  sourcemap: true,
+  target: 'es2016',
```

Comment:
Maybe we can bump this, what do you think?

Reply:
I'd say we either use `es6/es2015` - a kinda recommended standard for supporting most browsers, or we use any later version, but then there is no guarantee for support in all modern browsers.

Reply:
Our official stance is we support the latest 2 major versions of a browser. Other than Safari, this gives us a very modern base.

https://www.contentful.com/faq/about-contentful/#which-browsers-does-contentful-support

Reply:
And consumers can always transpile the library themselves if they want to support something else.

Reply:
According to [caniuse.com](https://caniuse.com/?feats=mdn-javascript_operators_optional_chaining%2Cmdn-javascript_operators_nullish_coalescing%2Cmdn-javascript_builtins_globalthis%2Ces6-module-dynamic-import%2Cbigint%2Cmdn-javascript_builtins_promise_allsettled%2Cmdn-javascript_builtins_string_matchall%2Cmdn-javascript_statements_export_namespace%2Cmdn-javascript_statements_import_meta) we can safely bump to es2020 – what do you think @bgutsol @massao ?

Reply:
ok 👍